### DISCUSSION TITLE
Trying to make a 1.14 mod, got an error: non-unique class target name mappings

### DISCUSSION THREAD

**Discussion by u9g** [FabricMC/fabric-api#2208] [Mod Dev Support] [answered]
gradle.properties:
```
# Done to increase the memory available to gradle.
org.gradle.jvmargs=-Xmx1G
# Fabric Properties
# check these on https://modmuss50.me/fabric.html
minecraft_version=1.14
yarn_mappings=1.14+build.21
loader_version=0.13.3
# Mod Properties
mod_version=1.0-SNAPSHOT
maven_group=dev.u9g
archives_base_name=minecrft-data-generator-server
# Dependencies
# check this on https://modmuss50.me/fabric.html
fabric_version=0.28.5+1.14
```
Error:
https://gist.github.com/u9g/e289e2437b6eddf4e2dc9b7adc940390

How can I fix this?

**Comment by modmuss50** [comment, marked as answer]
In your build.gradle make sure its not using the `:v2` mappings. There is a bug with the retro generated v2 mappings for 1.14 causing them to not work as expected.

`mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"`

Should become:

`mappings "net.fabricmc:yarn:${project.yarn_mappings}"`

**Reply by u9g** [reply]
Works great, thanks!

**Reply by u9g** [reply]
Okay wait, now I can't use any net.minecraft classes
![FAeeQIY](https://user-images.githubusercontent.com/43508353/167718149-b263b0bd-65ad-4a0d-bab9-528546455997.png)

**Reply by u9g** [reply]
False alarm, just had to run genSources
